kpi/utils/cache.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from functools import wraps
+
+from django_request_cache import get_request_cache
+
+
+def void_cache_for_request(keys):
+    """
+    Decorator that removes keys from to current request cache
+    Useful to void cache if content has to be refreshed within
+    the same request (e.g. on save() or delete())
+
+    :param keys: tuple. substrings to be searched for.
```

Comment:
I used `tuple` because (I think) `.startswith` needs a tuple to do the check on multiple strings at a time. 
You are right, they can be only prefixes of keys.